Blazor_Foundation_6_Test_Shared/Documentation/Containers/Off-Canvas.razor

```diff
@@ -1,6 +1,419 @@
+        </p>
+        <HighlightCS Language="html" Full="true" HeaderIcon="mdi-language-html5" HeaderBGcolor="#5d2b90" HeaderTitle="Razor"
+                     Content="PE9mZkNhbnZhc1dyYXBwZXI+CiAgPE9mZkNhbnZhcyBQb3NpdGlvbj0icG9zaXRpb24tbGVmdCIgSWQ9Im9mZkNhbnZhcyI+CiAgICA8IS0tIFlvdXIgbWVudSBvciBPZmYtY2FudmFzIGNvbnRlbnQgZ29lcyBoZXJlIC0tPgogIDwvT2ZmQ2FudmFzPgogIDxPZmZDYW52YXNDb250ZW50PgogICAgPCEtLSBZb3VyIHBhZ2UgY29udGVudCBsaXZlcyBoZXJlIC0tPgogIDwvT2ZmQ2FudmFzQ29udGVudD4KPC9PZmZDYW52YXNXcmFwcGVyPg==" />
+
+        <h2>Click Triggers</h2>
+        <p>
+            To create a click trigger that opens the panel, add the attribute <Code>data-open</Code> or <Code>DataToggle</Code> to any element. That element will then open or toggle the panel when clicked on. The value of the data attribute should be the ID of the off-canvas.
+        </p>
+        <HighlightCS Language="html" Full="true" HeaderIcon="mdi-language-html5" HeaderBGcolor="#5d2b90" HeaderTitle="Razor"
+                     Content="PEJ1dHRvbiBEYXRhVG9nZ2xlPSJvZmZDYW52YXMiPk9wZW4gTWVudTwvQnV0dG9uPg==" />
+        <Button DataToggle="offCanvas1">Open Menu</Button>
+
+        <h2>Close Button</h2>
+        <p>
+            Foundation's Close component can be used inside the off-canvas to close it.
+        </p>
+        <HighlightCS Language="html" Full="true" HeaderIcon="mdi-language-html5" HeaderBGcolor="#5d2b90" HeaderTitle="Razor"
+                     Content="PEJ1dHRvbiBDbGFzcz0iY2xvc2UtYnV0dG9uIiBkYXRhLWNsb3NlPgogIDxzcGFuIGFyaWEtaGlkZGVuPSJ0cnVlIj4mdGltZXM7PC9zcGFuPgo8L0J1dHRvbj4=" />
+
+        <h2>Complete Example</h2>
+        <p>
+            Here's a complete example that can be pasted into the <Code>body</Code> tag of your page. It includes a close button and basic menu styles.
+        </p>
+        <HighlightCS Language="html" Full="true" HeaderIcon="mdi-language-html5" HeaderBGcolor="#5d2b90" HeaderTitle="Razor"
+                     Content="PEJ1dHRvbiBEYXRhVG9nZ2xlPSJvZmZDYW52YXMiPk9wZW4gTWVudTwvQnV0dG9uPgo8T2ZmQ2FudmFzIFBvc2l0aW9uPSJwb3NpdGlvbi1sZWZ0IiBJZD0ib2ZmQ2FudmFzIj4KCiAgPCEtLSBDbG9zZSBidXR0b24gLS0+CiAgPEJ1dHRvbiBDbGFzcz0iY2xvc2UtYnV0dG9uIj4KICAgIDxzcGFuIGFyaWEtaGlkZGVuPSJ0cnVlIj4mdGltZXM7PC9zcGFuPgogIDwvQnV0dG9uPgoKICA8IS0tIE1lbnUgLS0+CiAgPHVsIGNsYXNzPSJ2ZXJ0aWNhbCBtZW51Ij4KICAgIDxsaT48YSBocmVmPSIjIj5CbGF6b3I8L2E+PC9saT4KICAgIDxsaT48YSBocmVmPSIjIj5Eb3Q8L2E+PC9saT4KICAgIDxsaT48YSBocmVmPSIjIj5Gb3VuZGF0aW9uPC9hPjwvbGk+CiAgICA8bGk+PGEgaHJlZj0iIyI+Q29tPC9hPjwvbGk+CiAgICA8bGk+PGEgaHJlZj0iIyI+U2xhc2g8L2E+PC9saT4KICAgIDxsaT48YSBocmVmPSIjIj5TaXRlczwvYT48L2xpPgogIDwvdWw+Cgo8L09mZkNhbnZhcz4KCjxPZmZDYW52YXNDb250ZW50PgogIDxoNT5Zb3VyIHBhZ2UgY29udGVudCBsaXZlcyBoZXJlPC9oNT4KPC9PZmZDYW52YXNDb250ZW50Pg==" />
+
+        <Button DataToggle="offCanvas">Open Menu</Button>
```

Comment:
Content of off-canvas is disproportionate

Reply:
```suggestion
                <HighlightCS Language="html" Full="true" HeaderIcon="mdi-language-html5" HeaderBGcolor="#5d2b90" HeaderTitle="Razor"
                     Content="PEJ1dHRvbiBEYXRhVG9nZ2xlPSJvZmZDYW52YXMiPk9wZW4gTWVudTwvQnV0dG9uPgo8T2ZmQ2FudmFzIFBvc2l0aW9uPSJwb3NpdGlvbi1sZWZ0IiBJZD0ib2ZmQ2FudmFzIj4KICA8IS0tIENsb3NlIGJ1dHRvbiAtLT4KICA8QnV0dG9uIENsYXNzPSJjbG9zZS1idXR0b24iPgogICAgPHNwYW4gYXJpYS1oaWRkZW49InRydWUiPiZ0aW1lczs8L3NwYW4+CiAgPC9CdXR0b24+CiAgPCEtLSBNZW51IC0tPgogIDx1bCBjbGFzcz0idmVydGljYWwgbWVudSI+CiAgICA8bGk+PGEgaHJlZj0iIyI+Qmxhem9yPC9hPjwvbGk+CiAgICA8bGk+PGEgaHJlZj0iIyI+RG90PC9hPjwvbGk+CiAgICA8bGk+PGEgaHJlZj0iIyI+Rm91bmRhdGlvbjwvYT48L2xpPgogICAgPGxpPjxhIGhyZWY9IiMiPkNvbTwvYT48L2xpPgogICAgPGxpPjxhIGhyZWY9IiMiPlNsYXNoPC9hPjwvbGk+CiAgICA8bGk+PGEgaHJlZj0iIyI+U2l0ZXM8L2E+PC9saT4KICA8L3VsPgo8L09mZkNhbnZhcz4KPE9mZkNhbnZhc0NvbnRlbnQ+CiAgPGg1PllvdXIgcGFnZSBjb250ZW50IGxpdmVzIGhlcmU8L2g1Pgo8L09mZkNhbnZhc0NvbnRlbnQ+" />

        <Button DataToggle="offCanvas">Open Menu</Button>
```